src/frontend/sharedTypes/navigation.ts

```diff
@@ -35,7 +35,7 @@ export type RootStackParamsList = {
   Settings: undefined;
   ProjectConfig: undefined;
   AboutMapeo: undefined;
-  LanguageSettings: undefined;
```

Comment:
nit: think keeping this as the navigation name is a little clearer and less prone to ambiguity in the future. not sure if there would be any other kind of language-related screens in the future but you never know 🤷